aptos-node/src/lib.rs

```diff
@@ -580,13 +583,30 @@ where
             genesis_config.allow_new_validators = true;
             genesis_config.epoch_duration_secs = EPOCH_LENGTH_SECS;
             genesis_config.recurring_lockup_duration_secs = 7200;
-            genesis_config.jwk_consensus_config_override = match env::var("INITIALIZE_JWK_CONSENSUS") {
+
+           match env::var("ENABLE_KEYLESS_DEFAULT") {
                 Ok(val) if val.as_str() == "1" => {
-                    let config = OnChainJWKConsensusConfig::default_enabled();
-                    println!("Flag `INITIALIZE_JWK_CONSENSUS` detected, will enable JWK Consensus for all default OIDC providers in genesis: {:?}", config);
-                    Some(config)
+                    println!("Flag `ENABLE_KEYLESS_DEFAULT` detected, will install default Groth16VerificationKey and OIDC providers");
```

Comment:
So, again, I don't us to have a default VK anymore in `aptos-core/`: it will tempt us to hard code proofs for that VK in our tests / benchmarks which is not sustainable.